packages/design-system-twrnc-preset/src/Theme/Theme.providers.tsx

```diff
@@ -2,19 +2,17 @@ import React, { createContext, useState, useMemo } from 'react';
 import { useColorScheme } from 'react-native';
```

Comment:
I noticed that mobile implements its own custom hook for theme detection. Ideally, the theme provider should delegate the decision between light and dark mode to the consuming platform, rather than trying to infer the OS theme internally. This approach would simplify the provider logic and improve maintainability. For reference, see how this is handled in https://github.com/MetaMask/metamask-mobile/blob/main/app/util/theme/index.ts#L77-L102